internal/runner.js

```diff
@@ -82,31 +83,31 @@ function compile(rawArgs) {
   const outCssMapPath =
       args.outCssMapFile ? path.join(cwd, args.outCssMapFile) : null;
 
-  // We use two parallel arrays of PostCSS plugin requires => strings of JS args.
+  // We use two parallel arrays, PostCSS plugin requires => strings of JS args.
   // To use in PostCSS, convert these into the actual plugin instances.
   const pluginRequires = argAsArray(args, 'pluginRequires');
   const pluginArgs = argAsArray(args, 'pluginArgs');
   const pluginInstances = pluginRequires.map(
       (nodeRequire, i) => {
         // Try and resolve this plugin as a module identifier.
-        let plugin;
-        try {
-          plugin = require(nodeRequire);
-        } catch { }
-
-        // If it fails, use the runfile helper in case it's a workspace file.
-        if (!plugin) {
-          try {
-            plugin = require(runfiles.resolve(nodeRequire));
-          } catch { }
-        }
+        let plugin;	
+        try {	
+          plugin = require(nodeRequire);	
+        } catch { }	
+
+        // If it fails, use the runfile helper in case it's a workspace file.	
+        if (!plugin) {	
+          try {	
+            plugin = require(runfiles.resolve(nodeRequire));	
+          } catch { }	
+        }	
 
-        // If that still fails, throw an error.
-        if (!plugin) {
-          const e = new Error(
-              `could not resolve plugin with node require ${nodeRequire}`);
-          e.code = 'MODULE_NOT_FOUND';
-          throw e;
+        // If that still fails, throw an error.	
+        if (!plugin) {	
+          const e = new Error(	
+              `could not resolve plugin with node require ${nodeRequire}`);	
+          e.code = 'MODULE_NOT_FOUND';	
+          throw e;	
```

Comment:
Looks like there's a bunch of trailing whitespace here.

Reply:
Done